src/psyclone/tests/dynamo0p3_test.py

```diff
@@ -3141,9 +3159,6 @@ def test_halo_dirty_4():
         "      !\n"
         "      ! Set halos dirty/clean for fields modified in the above loop\n"
         "      !\n"
-        "      CALL chi_proxy(1)%set_dirty()\n"
```

Comment:
I don't think halo exchanges should be affected by this PR. Please could you investigate what's happened to them.

Reply:
Hi @arporter. I removed them because chi is the coordinate field and is static, so never needs a halo exchange call on it. However, thinking about it, that is not what you are testing here, merely can Psyclone generate a halo exchange on a 3-field. I am reasonable confident that we only use the 3-field like this for static fields. I'll reinstate the tests.

Reply:
Done.